Engine/Module/ModuleRender.h

```diff
@@ -11,7 +11,11 @@
 #include <GL/glew.h>
 #include <list>
 
+class ComponentBillboard;
 class ComponentMeshRenderer;
+class ComponentParticleSystem;
+class ComponentTrail;
+class ComponentTrailRenderer;
```

Comment:
you can remove these fordward declarations

Reply:
alright